scipy/stats/_qmc.py

```diff
@@ -1117,14 +1117,16 @@ class Halton(QMCEngine):
     Pseudo-random number generator that generalize the Van der Corput sequence
     for multiple dimensions. The Halton sequence uses the base-two Van der
     Corput sequence for the first dimension, base-three for its second and
-    base-:math:`n` for its n-dimension.
+    base-:math:`n` for its :math:`n`-dimension, with :math:`n` the
+    :math:`n`'th prime.
```

Comment:
I think something like this is needed to disambiguate.

```suggestion
    base-:math:`p` for its :math:`n`-dimension, with :math:`p` the
    :math:`n`'th prime.
```